apps/backoffice-v2/src/domains/auth/components/UnauthenticatedLayout/UnauthenticatedLayout.layout.tsx

```diff
@@ -1,8 +1,7 @@
-import { useUnauthenticatedLayout } from './hooks/useUnauthenticatedLayout/useUnauthenticatedLayout';
 import { FunctionComponentWithChildren } from '../../../../common/types';
 
 export const UnauthenticatedLayout: FunctionComponentWithChildren = ({ children }) => {
-  useUnauthenticatedLayout();
+  // useUnauthenticatedLayout();
```

Comment:
Why not delete this line?

Reply:
I added a comment on that line for the future. I can document a reminder to uncomment the line or add a task to do it instead and remove the line instead. Was just a fear of this hook getting lost to the abyss.